ci-operator/step-registry/idp/openldap/idp-openldap-commands.sh

```diff
@@ -263,15 +288,16 @@ function handle_error() {
 # Check cluster type and skip steps if necessary
 if [ -f "${SHARED_DIR}/cluster-type" ] ; then
     CLUSTER_TYPE=$(cat "${SHARED_DIR}/cluster-type")
-    if [[ "$CLUSTER_TYPE" == "osd" ]] || [[ "$CLUSTER_TYPE" == "rosa" ]] || [[ "$CLUSTER_TYPE" == "hypershift-guest" ]]; then
+    if [[ "$CLUSTER_TYPE" == "osd" ]] || [[ "$CLUSTER_TYPE" == "rosa" ]]; then
```

Comment:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh (https://github.com/openshift/release/pull/53680/files) already made this. Looks like you need rebase on that latest code?

Reply:
I think changes you have mentioned is for `htpasswd` idp script. Changes i have done is for `openldap`  idp script. see [idp-openldap-commands.sh#L266](https://github.com/openshift/release/blob/master/ci-operator/step-registry/idp/openldap/idp-openldap-commands.sh#L266 )
if i understand your question correctly.